apps/site/assets/ts/schedule/components/schedule-finder/TableRow.tsx

```diff
@@ -61,6 +67,11 @@ const BusTableRow = ({
         onClick={onClick}
         onKeyPress={e => handleReactEnterKeyPress(e, onClick)}
       >
+        {anySchoolTrips && (
+          <td className="schedule-table__td--tiny">
+            <strong aria-label="school trip">{isSchoolTrip && "S"}</strong>
```

Comment:
I see your point about each row needing this element. I like the approach to check anySchoolTrips. 

I think we should also have two states for the inner content, so if this individual <td> is not a school trip, then there's no <strong></strong> element (or it has different content or label). I'm not sure what SR would say with an empty <strong></strong> element, though? so perhaps this is OK?

Reply:
I thought this was going to be more work, then I did it and it was easy :) are you cool w/ keeping this implementation?

Reply:
Please don't use the label or text "school trip".

Reply:
yes, we are removing this

Reply:
(sorry, the "keep this" was about something else, not the wording)

Reply:
I'm good with the way that you're only including that row if any of the trips are *supplemental service. 

For a service with some supplemental trips, we might need to make a tweak to the empty state for non-supplemental trips (maybe just moving isSchoolTrip && outside of the <strong>, or having the label say something different)